ykrt/src/compile/jitc_yk/codegen/reg_alloc/mod.rs

```diff
@@ -31,7 +31,12 @@ pub(crate) enum LocalAlloc {
     ///
     /// FIXME: unimplemented.
     Register,
-    ConstInt(u64),
+    /// A constant integer `bits` wide (see [jit_ir::Const::ConstInt] for the constraints on the
+    /// bit width) and with value `v`.
+    ConstInt {
+        bits: u32,
```

Comment:
I wonder if we should store a type index instead, then we can just compare the type indices in the well-formed check.

Reply:
The integer type only stores the bit size, so storing the `TyIdx` would just force us to look things up unnnecessarily. I *think* it's fine (and probably best) to store just the number of bits in the codegen.